src/filesystem/implementations/gcs.h

```diff
@@ -380,9 +381,12 @@ GCSFileSystem::LocalizePath(
         "GCS file localization not yet implemented " + path);
   }
 
+  // Create a local directory for GCS model store.
+  std::string env_mount_dir =
+      GetEnvironmentVariableOrDefault("TRITON_GCS_MOUNT_DIRECTORY", "/tmp");
```

Comment:
Can `"/tmp"` be some `common.h` constant like `constexpr char kDefaultMountDirectory[] = "/tmp"` or whatever name makes most sense?